src/jdk.jlink/share/classes/jdk/tools/jlink/internal/plugins/DefaultCompressPlugin.java

```diff
@@ -92,16 +93,29 @@ public void configure(Map<String, String> config) {
         if (level != null) {
             switch (level) {
                 case LEVEL_0:
+                    System.err.println(getMessage("warn.prefix") + " " +
```

Comment:
```suggestion
                    System.err.println(getMessage("compress.warn.argumentdeprecated", level));
```

It can be simpler to out the warning prefix in the message.  Same for line 102 and 107.

Reply:
I saw you updated plugins.properties but you haven't made this change.